src/Microsoft.Diagnostics.Monitoring.WebApi/EgressOutputOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    internal sealed class EgressOutputOptions : IEgressOutputOptions
```

Comment:
I'm trying to standardize the naming of our options classes. Anything that is a direct object model to a user settable option, should end in `*Options`. Anything that is a class we use internally to store options or configuration values should be named `*Configuration`.
If I understand this correctly this should be `EgressOutputConfiguration` and `IEgressOutputConfiguration`